.clang-format

```diff
@@ -0,0 +1,93 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
I used the file from https://github.com/torvalds/linux/blob/master/.clang-format as a starting point but eventually it will be replaced by our own, which will be similar to the one from linux kernel, but that is to be expected, as both files will be enforcing the K&R formatting.

Reply:
To paraphrase the [Linux kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html), I'd suggest downloading a copy, not reading it and deleting it as a symbolic gesture.